javascript/javascript1/week2/flight_booking.js

```diff
@@ -0,0 +1,27 @@
+function getFullname(firstname,surname,useFormalName,gender){
+    if(firstname==='' || surname===''){
+        console.log('Enter the valid name ?');
+  // prompt('enter the valid firstname and surname');
+    }
+    if(useFormalName){
+        return `${firstname} ${surname}`;
+    }
+
+    if(gender==='male'){
+        return `Lord ${firstname} ${surname}`;
+    }else if(gender==='female'){
+        return `Lady ${firstname} ${surname}`;
+    }
+return `Other ${firstname} ${surname}`;
+const fullname1=getFullname('','');
```

Comment:
I think there is something very wrong with the code here (function not terminated correctly) and indenting is also wonky.

Reply:
yes thanks a lot , there was a blunder mistake, without else section and without closing the function